public/app/features/admin/ldap/LdapPage.tsx

```diff
@@ -111,31 +108,32 @@ export class LdapPage extends PureComponent<Props, State> {
 
             {canReadLDAPUser && (
               <>
-                <h3 className="page-heading">Test user mapping</h3>
-                <div className="gf-form-group">
-                  <form onSubmit={this.search} className="gf-form-inline">
-                    <FormField
-                      label="Username"
-                      labelWidth={8}
-                      inputWidth={30}
-                      type="text"
-                      id="username"
-                      name="username"
-                      defaultValue={queryParams.username}
-                    />
-                    <Button type="submit">Run</Button>
-                  </form>
-                </div>
+                <h3>Test user mapping</h3>
+                <Form onSubmit={this.search}>
```

Comment:
Since native form changed to the Form component from the Grafana UI, handler should be updated as well in order to make button working properly. `onSubmit()` handler now accept `{ username: <value> }` -like parameter instead of event.
```suggestion
                <Form onSubmit={(data) => this.search(data)}>
```

`search()` method also should be adapted to use new data model instead of event.

Reply:
Thanks for much @alexanderzobnin for the help! I have made the code change, let me know if there is something else I need to fix or clean up!